README.md

```diff
@@ -1,47 +1,161 @@
+    <Button className='primary'>Close</Button>
+  </div>
+</Modal>
 ```
 
-## Use with css modules
+## Features
+
+React-e is a drop in replacement for `react.createElement`, but it provides syntactic advantages:
+
+#### `props` can be omitted even if you have children:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre>e('div', e('span'))</pre></td>
+    <td><pre>createElement('div', {}, createElement('span'))</pre></td>
+    <td><pre>&lt;div&gt;&lt;span&gt;&lt;/span&gt;&lt;/div&gt;</pre></td>
+  </tr>
+</table>
+
+#### static classnames can be provided inline using jade syntax:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre>e('div.foo.bar')</pre></td>
+    <td><pre>createElement('div', {className: 'foo bar'})</pre></td>
+    <td><pre>&lt;div className="foo bar"&gt;&lt;/div&gt;</pre></td>
+  </tr>
+</table>
 
-```coffeescript
-styles = require 'index.css'
-###
-styles =
-  container: 'local1'
-  primary: 'local2'
-  text: 'local3'
-###
+#### `div` is assumed if tag is not provided:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre>e('.foo')</pre></td>
+    <td><pre>createElement('div', {className: 'foo'})</pre></td>
+    <td><pre>&lt;div className="foo"&gt;&lt;/div&gt;</pre></td>
+  </tr>
+</table>
+
+#### [classnames](https://www.npmjs.com/package/classnames) module is built-in:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre>
+e('.foo', {
+  className: {active: this.state.isActive}
+})
+    </pre></td>
+    <td><pre>
+classnames = require('classnames')
+
+createElement('div', {
+  className: classnames('foo', {active: this.state.active})
+})
+    </pre></td>
+    <td><pre>
+classnames = require('classnames')
+
+&lt;div
+  className={classnames('foo', {active: this.state.active})}
+&gt;&lt;/div&gt;
+    </pre></td>
+  </tr>
+</table>
+
+#### [css modules](https://github.com/css-modules/css-modules) are bound when using `require('react-e/bind')(styles)`:
+<table>
+  <tr>
+    <th>react-e example</th>
+    <th>react.createElement example</th>
+    <th>JSX example</th>
+  </tr>
+  <tr>
+    <td><pre>
+styles = require('./stylesheet.css')
 e = require('react-e/bind')(styles)
 
-e '.container',
-  e 'button.primary', 'Click me'
-  e 'span.text', className: active: true, 'Some text'
-```
-Is equivalent to:
-```jsx
-<div className="local1">
-  <button className="local2">Click me</button>
-  <span className="local3 active">Some text</span>
-</div>
-```
+e('.foo', {className: someVar})
+    </pre></td>
+    <td><pre>
+styles = require('./stylesheet.css')
+
+createElement('div', {
+  className: styles.foo + ' ' + styles[someVar]
+})
```

Comment:
I wanted to document the actual behavior in case users weren't familiar with `classnames.bind`. It ends up being equivalent, it may not be a fair comparison here though